.github/workflows/pull_request.yml

```diff
@@ -95,21 +95,41 @@ jobs:
         run: cmake --build build/ -j $(nproc)
       - name: Nebula Server
         run: |
-          sudo curl -sSL https://get.docker.com/ | sh
-          pip3 install --upgrade pip
-          pip3 install setuptools
-          pip3 install docker-compose
-          docker network ls
-          mkdir tmp
-          pushd tmp
-          git clone -b external_network https://github.com/Shylock-Hg/nebula-docker-compose.git
-          pushd nebula-docker-compose/
-          sed -i "s/\bdocker_nebula-net\b/$(docker network ls | grep github_network | tr -s [:blank:] | cut -d ' ' -f 2)/g" docker-compose.yaml
-          docker-compose up -d
-          docker network ls
-          sleep 10
-          popd
-          popd
+          case ${{ matrix.os }} in
+          centos7)
+            set +e
+            for i in {0..10}; do
+              wget https://oss-cdn.nebula-graph.com.cn/package/v2-nightly/$(./date.py --day_diff=$i)/nebula-graph-$(./date.py --day_diff=$i)-nightly.el7.x86_64.rpm
```

Comment:
How about using docker yaml with different environment variables?

Reply:
Can't run docker the ci is in docker already.